sema/commons/glob/globbery.py

```
@@ -0,0 +1,88 @@
+    @param path: The path to check.
+    @param glob: The glob to match against.
+    @return: True if the path matches the glob.
+    """
+    ...  # implementation to come
+
+
+class GlobMatchVisitor(ABC):
+    """Abstract interface for visiting paths matched by a glob."""
+
+    @abstractmethod
+    def visitExcluded(self, path: Path, glob: str) -> Any:
```

Ruff (0.8.2)

Dynamically typed expressions (typing.Any) are disallowed in `visitExcluded`

(ANN401)